app/assets/stylesheets/top-bar.scss

```diff
@@ -0,0 +1,98 @@
+$grey: #cfcfcf;
+$leds: #f90;
+
+#top-bar {
+  height: 3em;
+  background-color: #333;
+  box-shadow: 0 2px 3px rgba(#111,0.5);
+  z-index: 10;
+
+  a {
+    color: white;
+  }
+
+  #app-name {
+    text-align: center;
+    top: -1em;
+
+    #app-title {
+      font-weight: bold;
```

Comment:
Properties should be ordered color, font-size, font-weight, letter-spacing, text-shadow